topologies/atedut_12.binding

```diff
@@ -1,149 +0,0 @@
-# proto-file: github.com/openconfig/featureprofiles/blob/main/topologies/proto/binding.proto
```

Comment:
Why was this deleted? There is still an atedut_12.testbed.

Reply:
Ah I also meant to delete the .testbed

In general are any of these shapes being used? I assumed not because of there not being an enum in metadata for 12

Reply:
Aha, there may be some vendors who are using the 12 ports in a one-off way to "scale" test beyond the 9 link default. Can we leave them around until we make the announcement to stop providing the testbed flag? It's also part of the README file, which may need to be updated in this PR:
https://github.com/openconfig/featureprofiles/blob/main/README.md?plain=1#L95